src/main/java/carRacing/CarRacing.java

```diff
@@ -37,16 +50,25 @@ public void startRacing(int racingCount) {
     private void excuteCycle() {
         for (Car car : cars) {
             car.moveByRandomInt(RandomIntUtil.getRandomInt());
+            if (maxScore < car.getPosition())
+                maxScore = car.getPosition();
         }
     }
 
-    private void enrollCars(int carCount) {
-        for (int i = 0; i < carCount; i++) {
-            this.cars.add(new Car());
-        }
+    private void enrollCars(String carNames) {
+        String[] carNameList = carNames.split(DELIMITER);
+        Arrays.stream(carNameList).forEach(carName -> this.cars.add(new Car(carName)));
     }
 
     public List<Car> getCars() {
         return cars;
     }
+
+    public void racingWinner() {
+        List<String> winnerCars = cars.stream().filter(car -> car.getPosition() >= maxScore)
```

Comment:
getter를 사용하여 Car 객체의 정보를 노출하는 방법보단 Car 객체에게 물어보면 어떨까요 :) ?
```suggestion
        List<String> winnerCars = cars.stream().filter(car -> car.isWinner(maxScore))
```